pyCGM_Single/pyCGM.py

```diff
@@ -51,34 +52,30 @@ def pelvisJointCenter(frame):
     Returns
     -------
     pelvis : array
-        Returns an array that contains the pelvis origin in a 1x3 array of xyz values,
-        which is then followed by a 4x1x3 array composed of the pelvis x, y, z
+        Returns an array that contains the pelvis origin in a :math:`1\\times3` array of xyz values,
+        which is then followed by a :math:`4 \\times 1 \\times 3` array composed of the pelvis x, y, z
```

Comment:
can you double check the need for the escape character? 

also I think in this case it is cleaner to see the 1x3 in regular text, we can save LaTeX for equations.